packages/flutter/test/widgets/transitions_test.dart

```diff
@@ -251,20 +252,25 @@ void main() {
 
     final RenderPositionedBox actualPositionedBox = tester.renderObject(find.byType(Align));
     expect(actualPositionedBox.heightFactor, 0.0);
+    expect(actualPositionedBox.widthFactor, 2.0);
 
     controller.value = 0.0;
     await tester.pump();
     expect(actualPositionedBox.heightFactor, 0.0);
+    expect(actualPositionedBox.widthFactor, 2.0);
 
     controller.value = 0.75;
     await tester.pump();
     expect(actualPositionedBox.heightFactor, 0.5);
+    expect(actualPositionedBox.widthFactor, 2.0);
 
     controller.value = 1.0;
     await tester.pump();
     expect(actualPositionedBox.heightFactor, 1.0);
+    expect(actualPositionedBox.widthFactor, 2.0);
   });
 
+
```

Comment:
nit: this extra line is not needed.

Reply:
Oops, I got it removed in [test: Test the size transition cross axis size](https://github.com/flutter/flutter/pull/134659/commits/ed2bbcfad93dd8c1f947a9e4b723d67e9e0760b2)